package.json

```diff
@@ -1,79 +1,102 @@
 {
-    "name": "chooser",
-    "version": "1.0.0",
-    "description": "The new and improved Creative Commons License Chooser",
-    "author": "akmadian <akmadian@gmail.com>",
-    "private": false,
-    "repository": {
-        "type": "git",
-        "url": "git+https://github.com/creativecommons/chooser.git"
+  "name": "chooser",
+  "version": "1.0.0",
+  "private": false,
+  "description": "The new and improved Creative Commons License Chooser",
+  "author": "akmadian <akmadian@gmail.com>",
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build && shx rm -rf ./docs && shx cp -r ./dist ./docs",
+    "test:unit": "vue-cli-service test:unit",
+    "test:e2e": "vue-cli-service test:e2e",
+    "lint": "vue-cli-service lint",
+    "i18n:report": "vue-cli-service i18n:report -v --src src/**/*.vue --locales src/locales/**.json -o output.json",
+    "lint:js": "vue-cli-service lint",
```

Comment:
This is a duplicate of 'lint' command. We can make the 'lint' command do both js and style linting.

Reply:
I'll remove stylelint commands and make the 'lint' command do both js and style linting.